### PR TITLE
Issue #SB-27891 fix: autocomplete dropdown design fix

### PR DESCRIPTION
**Pull request by rakeshSgr**

### CODE REVIEW

File: src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.spec.ts [hunk 1]
```diff
@@ -1,10 +1,10 @@
-import { async, ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
+import { async, ComponentFixture, TestBed, fakeAsync, tick,flush } from '@angular/core/testing';
 import { MaterialAutoCompleteComponent } from './material-auto-complete.component';
 import { NO_ERRORS_SCHEMA } from '@angular/core';
 import { configureTestSuite } from '@sunbird/test-util';
 import { AutocompletePipe } from './auto-complete-pipe'
 
-describe('MaterialAutoCompleteComponent', () => {
+fdescribe('MaterialAutoCompleteComponent', () => {
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.spec.ts:7]: please remove this f describe

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.spec.ts (hunk 1 above)
Comment (rakeshSgr) [reply src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.spec.ts]: removed

File: src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.html [hunk 2]
```diff
@@ -1,10 +1,10 @@
 <div class="sb-autocomp-dropdown">
   <mat-form-field class="sb-autocomp-dropdown__formfield">
-    <input matInput (click)="DisplayDropdown()" readonly name="selectedDpdwnInput"
+    <input matInput (focus)="DisplayDropdown()" readonly name="selectedDpdwnInput"
       [value]="_selectedDpdwnInput ? _selectedDpdwnInput : ''" />
   </mat-form-field>
   <div class="sb-autocomp-dropdown__list" *ngIf="displayDropdown">
-    <input matInput #autocompleteInput placeholder="Search" autofocus />
+    <input matInput #autocompleteInput placeholder="Search" name="search"/>
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.html:7]: why is the name removed?

File: src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.ts [hunk 3]
```diff
@@ -34,6 +34,8 @@ export class MaterialAutoCompleteComponent implements OnChanges {
     this.dropDownSelectedShow();
   }
 
+  @ViewChild("autocompleteInput", { static: false }) searchField: ElementRef;
```
Comment (rajeevsathish) [inline src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.ts:37]: I think the static:false is not needed in angular 9 please check

**Review by rajeevsathish — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.html (hunk 2 above)
Comment (rakeshSgr) [reply src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.html:7]: auto focus not working, so written code in .ts to make it work.

Comment (rakeshSgr) [reply src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.html:7]: so removed from HtmlTemplate

File: src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.ts (hunk 3 above)
Comment (rakeshSgr) [reply src/app/client/src/app/modules/shared/components/material-auto-complete/material-auto-complete.component.ts]: Yes, Updated

**Review by rajeevsathish — APPROVED** [review]

**Pull request merged**